Automattic/pocket-casts-android
# Referrals: add send guest pass screen

## Description

This adds send guest pass screen.

Ignore
- Animation in cards
- View not retained on rotation
- 0 guest passes state


## Testing Instructions
1. Login with a paid account
2. Go to Profile tab
3. Tap on gift icon
4. ✅ Notice that send guest pass screen is shown and matches Figma design (FVJPaWlRMlosEpsRhqxnSD-fi-2189_9583)

## Screenshots or Screencast 

<img width = 320 src="https://github.com/user-attachments/assets/90b4ec9e-e812-4286-871e-278a38d82d5b"/>

<img width = 500 src="https://github.com/user-attachments/assets/baa725aa-928b-4a78-8a9f-6aa857386ca4"/>

## Checklist
 
#### I have tested any UI changes...
